rpc/src/v1/helpers/dapps.rs

```diff
@@ -22,12 +22,18 @@ use v1::types::LocalDapp;
 pub trait DappsService: Send + Sync + 'static {
 	/// List available local dapps.
 	fn list_dapps(&self) -> Vec<LocalDapp>;
+	/// Refresh local dapps list
+	fn refresh_local_dapps(&self) -> bool;
 }
 
+/// TODO: Update this trait to a struct
 impl<F> DappsService for F where
-	F: Fn() -> Vec<LocalDapp> + Send + Sync + 'static
+	F: Fn(bool) -> (Vec<LocalDapp>, bool) + Send + Sync + 'static
```

Comment:
Why is this implementation even needed? Is it only for tests? If yes it would be best to follow the todo above.